meinberlin/apps/budgeting/api.py

```diff
@@ -243,12 +244,14 @@ def list(self, request, *args, **kwargs):
         permissions["view_support_count"] = user.has_perm(
             "meinberlin_budgeting.view_support", self.module
         )
-        permissions["view_vote_count"] = user.has_perm(
-            "meinberlin_budgeting.view_vote_count", self.module
+        permissions["view_rate_count"] = self.module.has_feature("rate", Proposal)
+        permissions["view_comment_count"] = self.module.has_feature("comment", Proposal)
+        permissions["view_votes_left"] = has_feature_active(
+            self.module, Proposal, "vote"
         )
-        permissions["view_votes_left"] = self._has_valid_token_in_session(
-            response
-        ) and user.has_perm("meinberlin_budgeting.add_vote", self.module)
+        permissions["view_vote_count"] = self.module.module_end < timezone.now()
+
+        response = super().list(request, args, kwargs)
         response.data["permissions"] = permissions
```

Comment:
@phillimorland So I added these permissions in my PR... Probably went wrong when rebasing?

Reply:
@Rineee err rebasing, i really have to get better at that, nice catch, will clean it up :)